libs/CovidDatasets.py

```diff
@@ -238,7 +266,9 @@ def get_beds_by_country_state(self, country, state):
 class JHUDataset(Dataset):
     # The date of the first JHU data snapshot.
     _FIRST_JHU_DATE = datetime.date(2020, 1, 22)
-    _JHU_URL_TEMPLATE = 'https://raw.githubusercontent.com/covid-projections/covid-data-public/master/data/cases-jhu/csse_covid_19_daily_reports/{}.csv'
+    if not local_public_data_populated:
```

Comment:
Can you run this somewhere else than in the class definition? It would surprise me if I imported this class and it started downloading data

Reply:
I agree. It should probably be a conditional call in each of the get_data functions (beds, population, timeseries) whichever one calls it first downloads the local copy

Reply:
Can make this change.

Reply:
I moved the url behind a getter. on first get it will be downloaded.